packages/opentelemetry-plugin-grpc-js/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@opentelemetry/plugin-grpc-js",
+  "private": true,
+  "version": "0.8.3",
+  "description": "OpenTelemetry @grpc/grpc-js automatic instrumentation package.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js",
+  "scripts": {
+    "test": "nyc ts-mocha -p tsconfig.json test/**/*.test.ts",
+    "tdd": "npm run test -- --watch-extensions ts --watch",
+    "clean": "rimraf build/*",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "precompile": "tsc --version",
+    "version:update": "node ../../scripts/version-update.js",
+    "compile": "npm run version:update && tsc -p .",
+    "prepare": "npm run compile"
+  },
+  "keywords": [
+    "opentelemetry",
+    "grpc",
+    "grpc-js",
+    "nodejs",
+    "tracing",
+    "profiling",
+    "plugin"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": "^8.13.0 || >=10.10.0"
```

Comment:
is this on purpose - as "normally" we have this only
```
"node": ">=8.0.0"
```

Reply:
oops forgot to add a self-review here. This is from [here](https://github.com/grpc/grpc-node/blob/931a7c3e9e2fafd6822019ba0d8bf1c92fdaa69d/packages/grpc-js/package.json#L8-L10). Actually thinking more about this, I think this can just go away. The plugin itself shouldn't have a node version requirement